scripts/override_dates_in_dist.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+set -euo pipefail
```

Comment:
This won't work when building on Windows.  If we can achieve the same thing in Java or groovy it would automatically be crossplatform. Downside is it's probably more verbose.

Reply:
A .bat version would be an alternative as well but would mean we're maintaining two versions of the script.

And it's probably also a big win to just have reproducible builds on UNIX-like systems.

Reply:
Hmm I didn't think of Windows. I will look into making it cross platform somehow.